misc/xml-to-record-converter/src/main/java/io/ballerina/xmltorecordconverter/XMLToRecordConverter.java

```diff
@@ -97,7 +97,8 @@ private XMLToRecordConverter() {}
 
     public static XMLToRecordResponse convert(String xmlValue, boolean isRecordTypeDesc, boolean isClosed,
                                               boolean forceFormatRecordFields,
-                                              String textFieldName, boolean withNameSpaces) {
+                                              String textFieldName, boolean withNameSpaces, boolean withoutAttributes,
```

Comment:
Why choose the name `withoutAttributes` instead of `withAttributes` ?

Reply:
I feel like, we are loosing consistency between `withNameSpaces` parameter name

Reply:
I added `withoutAttributes` because the default behaviour includes attributes, and the checkbox we will show in the UI will be labeled `Without Attributes` or `Remove Attributes`. I thought changing the option to `withAttributes` might confuse the tooling team.